src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ CHANGELOG
  * Dispatch `SwitchUserEvent` on `security.switch_user`
  * Deprecated `Argon2iPasswordEncoder`, use `SodiumPasswordEncoder` instead
  * Deprecated `BCryptPasswordEncoder`, use `NativePasswordEncoder` instead
+ * Added `DeauthenticatedEvent` dispatched in case the user has changed when trying to refresh it
```

Comment:
What is "it" here? The user?

Reply:
I think "it" refers to the Token here. What about "Added `DeauthenticatedEvent` dispatched in case the user has changed when trying to refresh the token"?